app/components/post_list/post/burn_on_read/unrevealed/unrevealed_burn_on_read_post.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+
+import {deletePost, revealBoRPost} from '@actions/remote/post';
+import Button from '@components/button';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import {PostModel} from '@database/models/server';
+import {getFullErrorMessage, isErrorWithStatusCode} from '@utils/errors';
+import {showBoRPostExpiredSnackbar} from '@utils/snack_bar';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    buttonBackgroundStyle: {
+        backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+        height: 56,
```

Comment:
interesting about the size here.. this will definitely make the post list jump up and down if the sizes do not match with the post content, which is probably 100% of the time.